mixer/pkg/runtime/handler/signature.go

```diff
@@ -40,7 +42,7 @@ func (s signature) equals(other signature) bool {
 }
 
 // calculateSignature returns signature given handler and array of dynamic or static instances
-func calculateSignature(handler hndlr, insts interface{}) signature {
+func calculateSignature(handler hndlr, handlerConn *v1beta1.Connection, insts interface{}) signature {
```

Comment:
Instead of passing in a new param, could you add Connection() function into handler interface?

Also could you add a unit test to cover the case where a dynamic handler has connection configured? Thanks!

Reply:
1. I have been hesitating about whether add a Connection() method to hndlr or add a new param to calc. function and I chose the latter one. Now I will add the Connection() method to the interface.
2. I'll do this on the next commit.